libs/sdk-ui-ext/src/insightView/InsightView.tsx

```diff
@@ -216,32 +216,34 @@ const InsightViewCore: React.FC<IInsightViewProps & WrappedComponentProps> = (pr
             {error && !isDataLoading && (
                 <InsightError error={convertError(error)} ErrorComponent={ErrorComponent} />
             )}
-            <div
-                className="insight-view-visualization"
-                // make the visualization div 0 height so that the loading component can take up the whole area
-                style={isLoadingShown ? { height: 0 } : undefined}
-            >
-                <InsightRenderer
-                    insight={insightResult}
-                    workspace={workspace}
-                    backend={backendWithTelemetry}
-                    colorPalette={colorPalette ?? colorPaletteResult}
-                    config={config}
-                    execConfig={execConfig}
-                    drillableItems={drillableItems}
-                    executeByReference={executeByReference}
-                    filters={filters}
-                    locale={locale || (workspaceSettingsResult?.locale as ILocale) || DefaultLocale}
-                    settings={workspaceSettingsResult}
-                    ErrorComponent={ErrorComponent}
-                    LoadingComponent={LoadingComponent}
-                    onDrill={onDrill}
-                    onError={handleError}
-                    onExportReady={onExportReady}
-                    onLoadingChanged={handleLoadingChanged}
-                    pushData={pushData}
-                />
-            </div>
+            {!isDataLoading && (
```

Comment:
I am not so sure about this change, why is it necessary? If loading, the div should already be height: 0 so it will not be visible. I am worried that mounting and unmounting of the InsightRenderer this way might cause issues.
There were fixes to this area like https://github.com/gooddata/gooddata-ui-sdk/commit/f1a4f0d570f2150627b2d5eade5c817d266abbf5 in the past